frontend/src/component/incomingWebhooks/IncomingWebhooksModal/IncomingWebhooksForm/useIncomingWebhooksForm.ts

```diff
@@ -99,8 +99,6 @@ export const useIncomingWebhooksForm = (incomingWebhook?: IIncomingWebhook) => {
             return false;
         }
 
-        // TODO call backend to check if token name is unique
-
```

Comment:
This is not needed since:
 - We only hit this when creating a new incoming webhook, so the token name will always be unique for that webhook (there are no other tokens);
 - Other times are covered by a real-time check here: https://github.com/Unleash/unleash/blob/c224d7dc4cf2659179531f6fed6c90444b0346fd/frontend/src/component/incomingWebhooks/IncomingWebhooksModal/IncomingWebhooksForm/IncomingWebhooksTokens/IncomingWebhooksTokensCreateDialog.tsx#L44-L45